src/components/SideBar.jsx

```diff
@@ -42,11 +48,58 @@ function SideBar(props) {
             })
           }
         >
-          <div className="profile">
+          <div className="sidebar_align">
             <AccountCircleIcon className="icon" /> <span>Profile</span>
           </div>
         </li>
       </ul>
+      <Dialog
```

Comment:
later we may put this on a separate component NewPost. I'll be doing some refactoring as well.

Reply:
I want to do this work !! So, Should I create a new Issue !! 😅